index.js

```diff
@@ -12,6 +12,10 @@ const discourse = require('discourse-node-api')({
 });
 
 async function main() {
+  if (api_key === '') {
+    throw 'No API key specified. See README.';
```

Comment:
If you don't mind just wrapping this message in the Error constructor that would be nicer :)